tft.py

```diff
@@ -0,0 +1,213 @@
+from cdragontoolbox.binfile import BinFile, BinHashBase, BinHashValue
+from json import JSONEncoder
+import json
+import mmap
+import copy
+import re
+import os
```

Comment:
Imports are preferably put in this order: standard lib, external libs (dependencies), current package.